lib/app/client.js

```diff
@@ -140,6 +140,9 @@ async function loadAsyncComponents (to, from, next) {
     next()
   } catch (err) {
     err = err || {}
+    if (/^Loading chunk (\d)+ failed\./.test(err.message) && process.client) {
```

Comment:
Is process.client necessary here?

Reply:
not really as i understand, was just trying to be super safe

Reply:
Indeed, we are already in app/client.js